### ISSUE TITLE
Testing using PureCN

### ISSUE THREAD

**Issue by waemm** [lima1/PureCN#136]
Hi Markus,

I am trying to create a minimal "system test" dataset for PureCN, I am having some issues doing this at present, selecting small regions of the genome just seem to decrease the number of SNPs too much or does not provide enough informative intervals. Just curious if you have a simple example or a bit of guidance here, I have found some test files in PureCN/inst/extdata but I assume these are mostly for your unit tests.

Thanks!
Warren

**Comment by lima1** [comment]
Hi Warren,

uh, it's been a very long time I created those toy examples. I think those are based on normal WES data.  So I think the trick is to simulate pretty clean data. Not sure if tools like bamsurgeon can do allele-specific copy number now. For the unit cases, at some point they ran for too long, so I filtered the toy example even further.

If simulation is not straightforward, pick you best samples (lowest log-ratio standard error, high purity, minimal heterogeneity) and then remove genes until you reach 1Mb. With 1Mb, the runtime should be less than 2-3mins.  Since some regions are better covered than others, you probably want to down sample by region, not completely random (keep genes in poorly covered regions so that the gaps don't become too large).

Markus
